docs/sources/developers/angular_deprecation/angular-plugins.md

```diff
@@ -121,6 +121,10 @@ Latest Version: v0.0.6 | Signature: Community | Last Updated: 2021
 
 > **Migration available - plugin superseded:** The original plugin only claims support for Grafana v4.x.x, it was replaced with a [new plugin](https://grafana.com/grafana/plugins/grafadruid-druid-datasource/) based on React.
 
+### [Cognite Data Fusion](https://grafana.com/grafana/plugins/cognitedata-datasource/)
+
+Latest Version: 3.0.0 | Signature: Commercial | Last Updated: 2023
```

Comment:
Latest version is 3.1.0:

https://grafana.com/grafana/plugins/cognitedata-datasource/

![immagine](https://github.com/grafana/grafana/assets/16373015/4a81e1aa-1be3-4721-96e7-b06851d294ba)

```suggestion
Latest Version: 3.1.0 | Signature: Commercial | Last Updated: 2023
```